examples/04-Retrieval-Model.ipynb

```diff
@@ -0,0 +1,1847 @@
+{
```

Comment:
<p><code>To download the training and test datasets visit Ali-CCP: Alibaba Click and Conversion Prediction at tianchi.aliyun.com.</code> I think there is something missing in this sentence.</p><p> </p><p><code>Exporting user (query) tower, item embeddings, and user and item features</code>  Can we add - why we want to export them? I would call this learning objective <code>Exporting the model for production systems</code>  - we can later explain the different components</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>sounds good.</p>